plugins/CoreHome/stylesheets/selector.less

```diff
@@ -3,15 +3,20 @@
   line-height: 0;
 
   span.title,
-  a.title {
+  a.title,
+  button.title {
     .font-default(11px, 12px);
     display: inline-block;
     width: 100%;
     padding: 10px 15px 11px 13px;
+    background-color: transparent;
+    border: none;
+
     white-space: nowrap;
     cursor: pointer;
     text-transform: uppercase;
     text-decoration: none;
+    text-align: center;
```

Comment:
ℹ️ Only useful on mobile. It’s the default value for button so I had to choose.

| before | after |
|---|---|
| <img width="414" height="289" alt="Image" src="https://github.com/user-attachments/assets/e321a700-85ec-4712-ad68-f16baa58dc2d" />| <img width="414" height="289" alt="Image" src="https://github.com/user-attachments/assets/8ed7cf48-6105-48f2-9615-dc51952ebea6" />|